### PR TITLE
Specify exactly one version conformance declaration required/allowed

### LINKED ISSUE #581: Objects containing multiple namaste files
I assume that it is invalid for an object to have more than one object conformance declaration, yes? Do you want to add language to that effect so that it can have an error code or should the generic `E001` just be used?

### PR DESCRIPTION
**Pull request by zimeon**
Fixes #581

### CODE REVIEW

File: draft/spec/index.html [hunk 1]
```diff
@@ -437,9 +437,9 @@ <h2>Object Structure</h2>
         <h2>Object Conformance Declaration</h2>
         <p>
             The OCFL specification version declaration <span id="E002">MUST</span> be formatted according to the
-            [[!NAMASTE]] specification.
-            It <span id="E003">MUST</span> be a file in the base directory of the OCFL Object Root giving the OCFL
-            version in the filename. The filename <span id="E004">MUST</span> conform to the pattern
+            [[!NAMASTE]] specification. There <span id="E003">MUST</span> be exactly one version declaration file in
+            the base directory of the OCFL Object Root giving the OCFL version in the filename.
```
Comment (pwinckles) [inline draft/spec/index.html:441]: Unrelated to the substance of this PR, should this be `<a>OCFL Object Root</a>` here?

**Review by pwinckles — APPROVED** [review]
👍

File: draft/spec/index.html (hunk 1 above)
Comment (zimeon) [reply draft/spec/index.html]: That would be consistent with the OCFL Storage Root sentence. Have added

File: draft/spec/index.html [hunk 2]
```diff
@@ -1052,8 +1052,9 @@ <h2>Root Structure</h2>
         <h2>Root Conformance Declaration</h2>
         <p>
           The OCFL version declaration <span id="E075">MUST</span> be formatted according to the [[!NAMASTE]]
-          specification. It <span id="E076">MUST</span> be a file in the base directory of the <a>OCFL Storage Root</a>
-          giving the OCFL version in the filename. The filename <span id="E077">MUST</span> conform to the pattern
+          specification. There <span id="E076">MUST</span> be exaclty one version declaration file in the base
```
Comment (awoods) [inline draft/spec/index.html:1055]: typo: `exaclty`

File: draft/spec/validation-codes.md [hunk 3]
```diff
@@ -84,7 +84,7 @@
 | E073 | 'Empty directories MUST NOT appear under a storage root.' | https://ocfl.io/draft/spec/#E073
 | E074 | 'Although implementations may require multiple OCFL Storage Roots - that is, several logical or physical volumes, or multiple "buckets" in an object store - each OCFL Storage Root MUST be independent.' | https://ocfl.io/draft/spec/#E074
 | E075 | 'The OCFL version declaration MUST be formatted according to the NAMASTE specification.' | https://ocfl.io/draft/spec/#E075
-| E076 | '[The OCFL version declaration] MUST be a file in the base directory of the OCFL Storage Root giving the OCFL version in the filename.' | https://ocfl.io/draft/spec/#E076
+| E076 | 'There must be exaclty one version declaration file in the base directory of the OCFL Storage Root giving the OCFL version in the filename.' | https://ocfl.io/draft/spec/#E076
```
Comment (awoods) [inline draft/spec/validation-codes.md:87]: typo: `exaclty`

**Review by awoods — CHANGES_REQUESTED** [review]
Looks good, pending resolution of typo.

File: draft/spec/validation-codes.md (hunk 3 above)
Comment (zimeon) [reply draft/spec/validation-codes.md]: Thanks! Fixed in [5992508](https://github.com/OCFL/spec/pull/585/commits/5992508e8b81f3942d2374de253a9cfe7353c097)

File: draft/spec/index.html (hunk 2 above)
Comment (zimeon) [reply draft/spec/index.html]: Thanks, fixed in [5992508](https://github.com/OCFL/spec/pull/585/commits/5992508e8b81f3942d2374de253a9cfe7353c097)

**Review by awoods — APPROVED** [review]

**Review by neilsjefferies — APPROVED** [review]

**Review by rosy1280 — APPROVED** [review]

**Pull request merged**